src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -585,20 +659,8 @@ static unique_ptr<Scene> _useBuildHelper(const SvgNode* node, const Box& vBox, c
 
         Matrix mViewBox = {1, 0, 0, 0, 1, 0, 0, 0, 1};
         if ((!mathEqual(width, vw) || !mathEqual(height, vh)) && vw > 0 && vh > 0) {
-            auto sx = width / vw;
-            auto sy = height / vh;
-            if (symbol.preserveAspect) {
-                if (sx < sy) sy = sx;
-                else sx = sy;
-            }
-
-            auto tvx = symbol.vx * sx;
-            auto tvy = symbol.vy * sy;
-            auto tvw = vw * sx;
-            auto tvh = vh * sy;
-            tvy -= (symbol.h - tvh) * 0.5f;
-            tvx -= (symbol.w - tvw) * 0.5f;
```

Comment:
here was a bug - these should be width and height instead of symbol.w and symbol.h